mne/viz/epochs.py

```diff
@@ -169,14 +169,14 @@ def plot_epochs_image(epochs, picks=None, sigma=0., vmin=None,
     if combine is not None:
         ts_args["show_sensors"] = False
 
-    if picks is None or not _is_numeric(picks):
-        picks = _picks_to_idx(epochs.info, picks)
-        if group_by is None:
-            logger.info("No picks and no groupby, showing the first five "
-                        "channels ...")
-            picks = picks[:5]  # take 5 picks to prevent spawning many figs
-    else:
-        picks = _picks_to_idx(epochs.info, picks)
+    picks = [picks] if isinstance(picks, str) or _is_numeric(picks) else picks
```

Comment:
I'd say `picks = [picks] if (isinstance(picks, str) or _is_numeric(picks)) else picks`

Reply:
I'm still -1 on adding any new conditionals, in favor of letting the existing triage function handle it and optionally return how it made the picks array if necessary

Reply:
@jona-sassenhagen's suggested change here would make the line >79 chars. But anyway this line needs to die, as per @larsoner.